q2cli/core/artifact_cache_global.py

```diff
@@ -58,7 +58,7 @@ def set_used_artifact_cache(args):
 
 
 def unset_used_artifact_cache():
-    """Set the USED_ARTIFACT_CACHE back to the None.
+    """Set the USED_ARTIFACT_CACHE back to None.
```

Comment:
I mean None is a singleton so `the None` works

Reply:
I did consider that while making the commit; however, I am not willing to go that hard